.github/workflows/migrations.yml

```diff
@@ -0,0 +1,68 @@
+name: migrations
```

Comment:
Not sure if it should be a separate workflow or another job, went with a new workflow for separation of concerns